documentation/usersguide/configmain.md

```diff
@@ -1181,9 +1175,7 @@ Using no delay is generally *not* recommended.
 
 Using no delay will cause all host checks to be scheduled for execution at the same time.
 
-More information on how to estimate how the inter-check delay affects host check scheduling can be found <a href="checkscheduling.html#host_inter_check_delay">here</a>.Values are as follows:
-
-* n = Don't use any delay - schedule all host checks to run immediately (i.e. at the same time!)
+	* n = Don't use any delay - schedule all host checks to run immediately (i.e. at the same time!)
```

Comment:
seems like some spaces/tabs have been sliped in.

Reply:
Whoops, fixed.